### ISSUE TITLE
Add functionality to add banner in HTML template

### ISSUE THREAD

**Issue by rgieseke** [manubot/rootstock#128]
It could be useful to have a simple way to add info text in a highly visible banner like "Work in progress" or "Published, peer-reviewed version at [...]" to the head of the HTML file with some simple config.

(From #127)

**Comment by dhimmel** [comment]
Thanks @rgieseke for opening an issue. For some context, @rgieseke implemented this for one of [his manuscripts](https://openclimatedata.github.io/global-emissions/v/6da74f156589cbd4299aeb5f273db74c722fc5ab/) in https://github.com/openclimatedata/global-emissions/commit/709e23dab6dbee42486118064e5dcc978abb059e. That banner looks like:

> ![banner](https://user-images.githubusercontent.com/1117703/42536508-477adde6-8460-11e8-8fa5-4d5c7dcfef5d.png)

The source to create that banner is:

```html
<h1 style="color: white; background-color: orange; text-align: center; border: 1px solid black; padding: 10px;">Unreviewed, work-in-progress ...</h1>
```
It reminds me a bit of the [PeerJ Preprint](https://peerj.com/preprints/3100/) Now Published banner:

> ![peerj-preprint-banner](https://user-images.githubusercontent.com/1117703/42536569-711a1608-8460-11e8-8c43-2283bab599ff.png)

I think these banners are a great idea (and they [seem to translate](https://openclimatedata.github.io/global-emissions/v/6da74f156589cbd4299aeb5f273db74c722fc5ab/manuscript.pdf) to PDF properly). Possibly we could design a few template banners, such as:

1. Work-in-progress on GitHub
2. Posted as a preprint at URL
3. Now published at URL

Any other ideas on potential broadly applicable banners? Next step is the best way to implement this. I think creating it from metadata.yaml is a good suggestion, using jinja2 logic.

**Comment by agitter** [comment]
Is this intended to replace the more subtle header text that appears under the title?  I thought that looked nice for deep review

![image](https://user-images.githubusercontent.com/3532898/42537170-bcc97500-8459-11e8-8dbf-366ffb9d3677.png)

**Comment by dhimmel** [comment]
> Is this intended to replace the more subtle header text that appears under the title? I thought that looked nice for deep review

It would replace that text according to my proposal, although it wouldn't be hard to change it back to how the Deep Review currently displays it. @agitter do you not think a banner could look nicer?

**Comment by dhimmel** [comment]
Here are additional PeerJ banners for inspiration:

![peerj](https://user-images.githubusercontent.com/1117703/42537713-b6782ae8-8463-11e8-8085-5944a5ce0263.png)

**Comment by agitter** [comment]
It's only my mild stylistic preference.  It like the inconspicuous text and the gray PeerJ example banners more than the attention-getting bold boxes.  But I'm not opposed to exploring banner designs and tweaking this element of the manuscript.

**Comment by vsmalladi** [comment]
I like the grey banners. But overall think this is a good idea to implement

**Comment by rgieseke** [comment]
Thanks for getting this discussions started. I think there are, depending on area of the paper and traditions of fields, maybe different needs for highlighting the state of a paper, so maybe the style could be configurable as well?

**Comment by vincerubinetti** [comment]
How does this look for a standard set of options:
![image](https://user-images.githubusercontent.com/8326331/53911349-24af9d80-4024-11e9-8624-055aee9bb836.png)

Here's the markdown for it:

```markdown
<div class="banner grey">
<i class="fas fa-scroll fa-lg"></i> **Grey Banner**<br>
useful for *general information* - [manubot.org](https://manubot.org/)
</div>

<div class="banner blue">
<i class="fas fa-info-circle fa-lg"></i> **Blue Banner**<br>
useful for *important information* - [manubot.org](https://manubot.org/)
</div>

<div class="banner orange">
<i class="fas fa-ban fa-lg"></i> **Orange Banner**<br>
useful for *warnings* - [manubot.org](https://manubot.org/)
</div>

<div class="banner yellow">
<i class="fas fa-exclamation-triangle fa-lg"></i> **Yellow Banner**<br>
useful for *notifications* - [manubot.org](https://manubot.org/)
</div>

<div class="banner green">
<i class="fas fa-check-circle fa-lg"></i> **Green Banner**<br>
useful for indicating *success* - [manubot.org](https://manubot.org/)
</div>

<div class="banner red">
<i class="fas fa-times-circle fa-lg"></i> **Red Banner**<br>
useful for indicating *failure* - [manubot.org](https://manubot.org/)
</div>

<div class="banner purple">
<i class="fas fa-map-pin fa-lg"></i> **Purple Banner**<br>
useful for *special information* - [manubot.org](https://manubot.org/)
</div>
```

The `<i class="fas">` stuff is referencing Font Awesome, a webfont containing thousands of useful glyphs for icons. This is the cleanest way I could think of to allow flexible icons. The webfont has to be imported at the top of the theme like this: 

```CSS
@import url("https://use.fontawesome.com/releases/v5.7.2/css/all.css");
```
It will add load time to the page.

**Comment by vincerubinetti** [comment]
Implemented in [pr #178](https://github.com/manubot/rootstock/pull/178) and demo'd as follows in [pr #169](https://github.com/manubot/rootstock/pull/169):

![image](https://user-images.githubusercontent.com/8326331/54204749-450bac00-44ab-11e9-94e5-d311a726db24.png)

**Comment by dhimmel** [comment]
Closed by https://github.com/manubot/rootstock/pull/178. 

I like the new simple syntax, enabled by the new CSS classes:

```markdown
[Making banners!]{.banner .lightblue}
```

@rgieseke hopefully this addresses your needs.
